src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -533,7 +557,7 @@ def _job_settings(self, step_name: str, remote_wheel: str):
                 on_success=[self._my_username], on_failure=[self._my_username]
             )
         job_tasks = []
-        job_clusters: set[str] = set()
+        job_clusters: set[str] = {'main'}
```

Comment:
Maybe refer to default like:
```suggestion
        job_clusters: set[str] = {Task.job_cluster}
```